setup.py

```diff
@@ -59,7 +59,7 @@ def run(self):
         if self.type in ['unit', 'all']:
             cmd.append('Ganga/new_tests/*.py')
         if self.type in ['integration', 'all']:
-            cmd.append('Ganga/new_tests/GPI --testmatch="(?:\\b|_)([Tt]est|Savannah|JIRA)"')
+            cmd.append('Ganga/new_tests/GPI --testmatch="(?:\\b|_)(Test|Savannah|JIRA)"')
```

Comment:
I think in fact we need to keep this as lower case `t` as well so that it can match the function names correctly. Let's keep this line as it was.